sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/DocumentHelper.cs

```diff
@@ -0,0 +1,164 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+using System.Globalization;
+using System.Runtime.CompilerServices;
+using Azure.Monitor.OpenTelemetry.Exporter.Internals;
+using Azure.Monitor.OpenTelemetry.LiveMetrics.Models;
+
+namespace Azure.Monitor.OpenTelemetry.LiveMetrics.Internals
+{
+    /// <summary>
+    /// Helper class used to convert Activity to the LiveMetrics document schema.
+    /// </summary>
+    internal static class DocumentHelper
+    {
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal static RemoteDependency ConvertToRemoteDependency(Activity activity)
+        {
+            string urlFull = string.Empty, httpResponseStatusCode = string.Empty;
+
+            foreach (ref readonly var tag in activity.EnumerateTagObjects())
```

Comment:
This loop should stop early if we could get the values earlier, no need to traverse till the end.

Reply:
This was removed in favor of ActivityTagsProcessor.

Reply:
Please add a TODO to validate if we can create minimal version of ActivityTagsProcessor for live metric needs.